packages/plugins/plugin-navtree/src/components/L0Menu.tsx

```diff
@@ -287,26 +289,22 @@ const L0Collection = ({ item, path, parent }: L0ItemProps) => {
   );
 };
 
-export const L0Menu = ({
-  topLevelItems,
-  pinnedItems,
-  userAccountItem,
-  parent,
-  path,
-}: {
+export type L0MenuProps = {
   topLevelItems: Node<any>[];
   pinnedItems: Node<any>[];
   userAccountItem: Node<any>;
   parent?: Node<any>;
   path: string[];
-}) => {
+};
+
+export const L0Menu = ({ topLevelItems, pinnedItems, userAccountItem, parent, path }: L0MenuProps) => {
   return (
     <Tabs.Tablist classNames='group/l0 absolute z-[1] inset-block-0 inline-start-0 rounded-is-lg grid grid-cols-[var(--l0-size)] grid-rows-[1fr_min-content_var(--l0-size)] contain-layout !is-[--l0-size] bg-baseSurface border-ie border-separator app-drag  pbe-[env(safe-area-inset-bottom)]'>
       <ScrollArea.Root>
         <ScrollArea.Viewport>
           <div
             role='none'
-            className='grid auto-rows-[calc(var(--l0-size)-.5rem)] plb-1 [body[data-platform="darwin"]_&]:pbs-[calc(30px+0.25rem)] [body[data-platform="ios"]_&]:pbs-[max(env(safe-area-inset-top),0.25rem)]'
+            className='grid auto-rows-[calc(var(--l0-size)-.5rem)] [body[data-platform="darwin"]_&]:pbs-[calc(30px+0.25rem)] [body[data-platform="ios"]_&]:pbs-[max(env(safe-area-inset-top),0.25rem)]'
```

Comment:
Please ensure any spacing adjustments in DnD elements doesn't break their functionality.

Reply:
can you clarify? i think we should split super long class defs for legibility and grouping; that shouldn't affect the funcitonality?

Reply:
You removed `plb-1` which will change this DnD element's internal spacing, which has caused issues before.